cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -261,14 +262,21 @@ func startControllers(c *cloudcontrollerconfig.CompletedConfig, kubeconfig *rest
 	// If apiserver is not running we should wait for some time and fail only then. This is particularly
 	// important when we start apiserver and controller manager at the same time.
 	err = wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
-		if _, err = restclient.ServerAPIVersions(kubeconfig); err == nil {
-			return true, nil
+		healthStatus := 0
```

Comment:
Do we have the same code in kube-controller-manager? I would like to have this function in cmd/controller-manager as a generic helper.

Reply:
I will check.

Reply:
Yes, also in kube-controller-manager.